packages/components/app/styles/components/code-block/theme.scss

```diff
@@ -0,0 +1,156 @@
+  // interactive ---
+  --hds-code-block-color-foreground-interactive: #dedfe3;
+  --hds-code-block-color-foreground-interactive-hover: #fff;
+  --hds-code-block-color-foreground-interactive-active: #f1f2f3;
+  // actions ---
+  --hds-code-block-color-foreground-action-default: #2b89ff;
+  --hds-code-block-color-foreground-action-hover: #389aff;
+  --hds-code-block-color-foreground-action-active: #4ca1ff;
+  // borders
+  --hds-code-block-color-border-strong: rgba(178, 182, 189, 40%); // #b2b6bd66
+  --hds-code-block-color-border-primary: rgba(178, 182, 189, 20%); // #b2b6bd33
+
+  // CodeBlock UI:
+  // lines of code - highlighted ----
+  --hds-code-block-color-line-highlight: rgba(0, 74, 222, 20%);
+  --hds-code-block-color-line-highlight-border: #1555d4;
+
+  // Syntax highlighting tokens:
+  // general ----
+  --hds-code-block-color-token: var(--hds-code-block-color-matrix-green);
+  // specific ----
+  --hds-code-block-color-atrule: var(--hds-code-block-color-blue);
+  --hds-code-block-color-code-block-attr-name: var(--hds-code-block-color-blue);
+  --hds-code-block-color-attr-value: var(--hds-code-block-color-blue);
+  --hds-code-block-color-boolean: var(--hds-code-block-color-purple);
+  --hds-code-block-color-builtin: var(--hds-code-block-color-orange);
+  --hds-code-block-color-char: var(--hds-code-block-color-orange);
+  --hds-code-block-color-class-name: var(--hds-code-block-color-matrix-green);
+  --hds-code-block-color-comment: var(--hds-code-block-color-foreground-faint);
+  --hds-code-block-color-control: var(--hds-code-block-color-cyan);
+  --hds-code-block-color-constant: var(--hds-code-block-color-purple);
+  --hds-code-block-color-deleted: var(--hds-code-block-color-red);
+  --hds-code-block-color-entity: var(--hds-code-block-color-matrix-green);
+  --hds-code-block-color-function: var(--hds-code-block-color-blue);
+  --hds-code-block-color-important: var(--hds-code-block-color-red);
+  --hds-code-block-color-keyword: var(--hds-code-block-color-matrix-green);
+  --hds-code-block-color-namespace: var(--hds-code-block-color-red);
+  --hds-code-block-color-number: var(--hds-code-block-color-purple);
+  --hds-code-block-color-operator: var(--hds-code-block-color-cyan);
+  --hds-code-block-color-prolog: var(--hds-code-block-color-foreground-primary);
+  --hds-code-block-color-property: var(--hds-code-block-color-blue);
+  --hds-code-block-color-punctuation: var(--hds-code-block-color-foreground-primary);
+  --hds-code-block-color-regex: var(--hds-code-block-color-orange);
+  --hds-code-block-color-script: var(--hds-code-block-color-foreground-primary);
+  --hds-code-block-color-selector: var(--hds-code-block-color-matrix-green);
+  --hds-code-block-color-string: var(--hds-code-block-color-orange);
+  --hds-code-block-color-symbol: var(--hds-code-block-color-orange);
+  --hds-code-block-color-tag: var(--hds-code-block-color-matrix-green);
+  --hds-code-block-color-url: var(--hds-code-block-color-cyan);
+  // language-scoped:
```

Comment:
`css`, `javascript` and `markup/html` are actually included (not by us, but by Prism itself). we currently use JavaScript as part of the showcase. Happy to remove them, but I thought if the work is already done I might as well keep it in case we need it in future.